pkg/netpol/eval/resources.go

```diff
@@ -299,7 +313,7 @@ func (pe *PolicyEngine) GetPeersList() ([]Peer, error) {
 		res[index] = workloadPeer
 		index++
 	}
-	return res, nil
+	return res, evalErr
 }
 
 // GetDisjointIPBlocks returns a slice of disjoint ip-blocks from all netpols resources
```

Comment:
Why `GetDisjointIPBlocks` func is exported? (I didn't find references outside eval)

Reply:
#132